pkg/lwip/Makefile.dep

```diff
@@ -42,6 +42,7 @@ endif
 
 ifneq (,$(filter lwip_sock_%,$(USEMODULE)))
   USEMODULE += lwip_sock
+  CFLAGS += -DLWIP_SO_RCVTIMEO
```

Comment:
Would preferred this switch to be in `pkg/lwip/include/lwipopts.h`. Could this maybe be done as a follow-up?